core/config/default-config.js

```diff
@@ -531,6 +532,7 @@ const defaultConfig = {
         {id: 'screenshot-thumbnails', weight: 0, group: 'hidden'},
         {id: 'final-screenshot', weight: 0, group: 'hidden'},
         {id: 'script-treemap-data', weight: 0, group: 'hidden'},
+        {id: 'entity-classification', weight: 0, group: 'hidden'},
```

Comment:
The audit should still run even if it's not in a category. I don't think it makes sense to categorize this audit.

Reply:
Update, this is only true for `full-page-screenshot`, but we should add a special case for this too.